src/test/frontend/db/query_dsl_test.cljs

```diff
@@ -319,7 +334,7 @@ last-modified-at:: 1609084800002"}]]
       "(not [[page 1]])"
       {:query '([?b :block/uuid]
                 (not [?b :block/path-refs [:block/name "page 1"]]))
-       :count 36}))
+       :count 39}))
```

Comment:
These counts aren't helpful as they are too large to make any sense of it. I ran out of time to clean up but one day I'll make this easier to read